repair/row_level.cc

```diff
@@ -1270,6 +1270,8 @@ class repair_meta {
         // Repair rows in row_diff will be flushed to disk by flush_rows_in_working_row_buf,
         // so we skip calling do_apply_rows here.
         _dirty_on_master = is_dirty_on_master::yes;
+        // Clear gently to avoid stalls
+        utils::clear_gently(row_diff).get();
```

Comment:
I imagine this is a much larger stall using RBNO, because we'll have have large row_diff:s.

Reply:
I mean that RBNO bootstrap is different from regular repair, because 100% of the rows will be transferred. In this case smaller rows are worse, because there are more of them per repair buffer.

Reply:
Yes, yes. By "the bigger the rows",  I meant the number of rows not the size of the rows ;-)